source/usage-examples/bulkWrite.txt

```diff
@@ -92,15 +91,17 @@ The output should look something like this:
 .. include:: /includes/legacy-redirect.rst
 
 For additional information on the classes and methods mentioned on this
-page, see the following API documentation:
-
-- :java-sync-api:`bulkWrite <com/mongodb/client/MongoCollection.html#bulkWrite(java.util.List,com.mongodb.client.model.BulkWriteOptions)>`
-- :java-docs:`BulkWriteOptions <apidocs/mongodb-driver-core/com/mongodb/client/model/BulkWriteOptions.html>`
-- :java-docs:`BulkWriteResult <apidocs/mongodb-driver-core/com/mongodb/bulk/BulkWriteResult.html>`
-- :java-docs:`InsertOneModel <apidocs/mongodb-driver-core/com/mongodb/client/model/InsertOneModel.html>`
-- :java-docs:`UpdateOneModel <apidocs/mongodb-driver-core/com/mongodb/client/model/UpdateOneModel.html>`
-- :java-docs:`UpdateManyModel <apidocs/mongodb-driver-core/com/mongodb/client/model/UpdateManyModel.html>`
-- :java-docs:`DeleteOneModel <apidocs/mongodb-driver-core/com/mongodb/client/model/DeleteOneModel.html>`
-- :java-docs:`DeleteManyModel <apidocs/mongodb-driver-core/com/mongodb/client/model/DeleteManyModel.html>`
-- :java-docs:`ReplaceOneModel <apidocs/mongodb-driver-core/com/mongodb/client/model/ReplaceOneModel.html>`
+page, see the following resources:
+
+- :manual:`Unique Index </core/index-unique>` Server Manual Entry
+- :manual:`Schema Validation </core/schema-validation>` Server Manual Entry
```

Comment:
Comment: This is the first instance of changing manual links to an end of section I have seen in this PR. Apologies for thinking this was a wider reaching theme in this PR than it was in reality. I panicked when I thought this PR made this change throughout the whole documentation, and I didn't think clearly. I will try and not have this alarmist knee-jerk reaction in the future.